package.json

```diff
@@ -1,18 +1,20 @@
 {
         "name": "mako",
         "description": "Storage backend for Manta",
-        "version": "1.0.1",
+        "version": "1.0.2",
         "author": "Joyent (joyent.com)",
         "license": "MPL-2.0",
         "private": true,
         "dependencies": {
-                "assert-plus": "0.1.1",
-                "bunyan": "0.21.4",
+                "assert-plus": "1.0.0",
+                "bunyan": "1.8.12",
                 "carrier": "0.1.7",
-                "manta": "2.0.7",
-                "moray": "git+https://github.com/joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
+                "manta": "5.2.1",
+                "moray": "3.7.0",
                 "posix-getopt": "1.0.0",
-                "vasync": "1.6.4"
+                "restify": "4.3.4",
+                "triton-metrics": "1.0.2",
```

Comment:
Why are restify and triton-metrics being added? I don't see what is using those in manta-mako.git

Reply:
Sorry, that looks like stuff we only needed in ./bin/garbage-deleter.js in the master (mantav2) branch. I'll remove these.